lib/discordrb.rb

```diff
@@ -32,15 +32,19 @@ module Discordrb
     server_message_typing: 1 << 11,
     direct_messages: 1 << 12,
     direct_message_reactions: 1 << 13,
-    direct_message_typing: 1 << 14
+    direct_message_typing: 1 << 14,
+    message_content: 1 << 15,
+    server_scheduled_events: 1 << 16,
+    auto_moderation_configuration: 1 << 20,
+    auto_moderation_execution: 1 << 21
```

Comment:
If it's not too much trouble, could you please add the following new intents as well: 
`server_message_polls: 1 << 24`
`direct_message_polls: 1 << 25`